### PR TITLE
UnigramDev.Unigram version 8.9.7682.0

### PR DESCRIPTION
**Pull request by TheBossMagnus**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

### CODE REVIEW

File: manifests/u/UnigramDev/Unigram/8.9.7682.0/UnigramDev.Unigram.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+PackageIdentifier: UnigramDev.Unigram
+PackageVersion: 8.9.7682.0
+Installers:
+- Platform:
+  - Windows.Universal
+  MinimumOSVersion: 10.0.18362.0
+  Architecture: x64
+  InstallerType: msix
+  InstallerUrl: https://github.com/UnigramDev/Unigram/releases/download/latest/Unigram.msixbundle
```
Comment (Trenly) [inline manifests/u/UnigramDev/Unigram/8.9.7682.0/UnigramDev.Unigram.installer.yaml:12]: Please use a versioned URL and not a URL to Latest

**Review by Trenly — CHANGES_REQUESTED** [review]

File: manifests/u/UnigramDev/Unigram/8.9.7682.0/UnigramDev.Unigram.installer.yaml (hunk 1 above)
Comment (TheBossMagnus) [reply manifests/u/UnigramDev/Unigram/8.9.7682.0/UnigramDev.Unigram.installer.yaml:12]: they update a github release without creating a new one(https://github.com/UnigramDev/Unigram/releases/tag/latest)

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Package has dependency on the latest version of MUX 2.7:
```
2022-10-05 12:43:19.269 [CORE] Deployment operation #0: Windows cannot install package TelegramFZ-LLC.Unigram_8.9.7682.0_x64__1vfw5zm9jmzqy because this package depends on a framework that could not be found. Provide the framework "Microsoft.UI.Xaml.2.7" published by "CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US", with neutral or x64 processor architecture and minimum version 7.2203.17001.0, along with this package to install. The frameworks with name "Microsoft.UI.Xaml.2.7" currently installed are: {Microsoft.UI.Xaml.2.7_7.2109.13004.0_x64__8wekyb3d8bbwe}
2022-10-05 12:43:19.269 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\Deployment.cpp(159)\WindowsPackageManager.dll!00007FF97DB4990F: (caller: 00007FF97DABFA5E) Exception(1) tid(2294) 80073CF3 Package failed updates, dependency or conflict validation.
```

Azure Pipelines has `7.2109.13004.0` but Unigram requires `7.2203.17001.0`

File: manifests/u/UnigramDev/Unigram/8.9.7682.0/UnigramDev.Unigram.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: UnigramDev.Unigram
+PackageVersion: 8.9.7682.0
+PackageLocale: en-US
+Publisher: UnigramDev
+PackageName: Unigram
```
Comment (ItzLevvie) [inline manifests/u/UnigramDev/Unigram/8.9.7682.0/UnigramDev.Unigram.locale.en-US.yaml:8]:
```suggestion
PackageName: Unigram—Telegram for Windows
```
This must match Control Panel.

File: manifests/u/UnigramDev/Unigram/8.9.7682.0/UnigramDev.Unigram.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: UnigramDev.Unigram
+PackageVersion: 8.9.7682.0
+PackageLocale: en-US
+Publisher: UnigramDev
```
Comment (ItzLevvie) [inline manifests/u/UnigramDev/Unigram/8.9.7682.0/UnigramDev.Unigram.locale.en-US.yaml:7]:
```suggestion
Publisher: Telegram FZ-LLC
```
This must match Control Panel.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request closed without merging**